pkg/ddevapp/global_dotddev_assets/commands/host/shells/ddev.fish

```diff
@@ -12,7 +12,7 @@ function ddevcd
 end
 
 function __ddevcd_autocomplete
-  DDEV_VERBOSE=false ddev debug cd --list
+  DDEV_VERBOSE=false ddev debug cd --list 2>/dev/null
```

Comment:
Should we have `DDEV_DEBUG=false` too, just for you?

Reply:
Although it is not needed now, it is still good to add it.